bin/migrate-oats-data/documents/post_launch/oats_documents_to_alcs_documents_srw.py

```diff
@@ -181,7 +181,7 @@ def document_clean(conn=None):
     logger.info("Start documents cleaning")
     with conn.cursor() as cursor:
         cursor.execute(
-            f"DELETE FROM alcs.document WHERE audit_created_by = '{OATS_ETL_USER}' AND audit_created_at > '2024-02-08';"
+            f"DELETE FROM alcs.document WHERE audit_created_by = '{OATS_ETL_USER}' AND audit_created_at > '2024-02-08' AND oats_application_id IS NOT NULL;"
```

Comment:
I think this will delete all files, not only SRW documents

Reply:
in theory the only documents imported past that date that have an application_id should be SRWs, but yes it is not exclusive to SRWs